### PR TITLE
Relationship flowchart

### LINKED ISSUE #18: Create Relationship section in glossary
- Define sequential order of relationships 
- attributes of each relationship (are they all the same)

### PR DESCRIPTION
**Pull request by rmarow**
resolves #18

### CODE REVIEW

**Review by MattF-NSIDC — CHANGES_REQUESTED** [review]
Mermaid code should be included directly in the markdown!

File: glossary.md [hunk 1]
```diff
@@ -94,6 +94,16 @@ instance: Imagine a satelite (`observing system`) that is very important to a se
...
 `performance`) line. 
 
+
+```mermaid
+flowchart LR
+    A[Observing Systems]--> |supported by| B[Data Products]
+    A --> |self| A
+    B --> |supported by| C(Applications)
+    C --> C
+    C --> |supported by| D[Societal Benefit Areas]
```
Comment (rmarow) [inline glossary.md:104]: @hazelshapiro  you are able to edit here if you'd like.

Comment (MattF-NSIDC) [reply glossary.md:104]: I agree with @hazelshapiro 's comments on the verb and arrow direction. If we use `supported by`, the arrow should go to the left, from the subject to the object. If we use `support` or `supports`, then the subject and object are reversed and the arrow should go right.

The latter seems more intuitive to me!

Comment (rmarow) [reply glossary.md]: agreed, implemented that on the latest commit.

**Comment by MattF-NSIDC** [comment]
@hazelshapiro if you didn't already know, use this button to see the rich preview version :) 
image

**Review by MattF-NSIDC — APPROVED** [review]

File: glossary.md [hunk 2]
```diff
@@ -94,6 +94,16 @@ instance: Imagine a satelite (`observing system`) that is very important to a se
...
 
+
+```mermaid
+flowchart LR
+    A[Observing Systems]--> |supports| B[Data Products]
+    A --> |self| A
+    B --> |supports| C(Applications)
+    C --> C
+    C --> |supports| D[Societal Benefit Areas]
+```
```
Comment (rmarow) [inline glossary.md:105]: @hazelshapiro you can change or request changes here :)

Comment (hazelshapiro) [reply glossary.md:105]: Needs a B --> B link and then it'll be final.

**Pull request merged**